client/components/tinymce/index.jsx

```diff
@@ -260,12 +261,14 @@ export default class extends React.Component {
 		const store = this.context.store;
 		let isRtl = false;
 		let localeSlug = 'en';
+		let colorScheme = undefined;
```

Comment:
Is there a reason why you assign `undefined` here? Isn't it the same as `let colorScheme;`?